spec/rubocop/cop/style/redundant_parentheses_spec.rb

```diff
@@ -793,6 +793,51 @@ def foo
     RUBY
   end
 
+  it 'accepts parentheses in `return` with multiline style argument' do
+    expect_no_offenses(<<~RUBY)
+      return (
+        42
+      )
+    RUBY
+  end
+
+  it 'registers an offense when parentheses in `return` with single style argument' do
```

Comment:
Should we also have a test for `(return 42)`?

Reply:
It seems to be a false negative, and it seems like `(x 42)` is also worked in the same. I'd like to address these false negatives collectively in a separate PR.

Reply:
Fine by me.